src/index.html

```diff
@@ -11,7 +11,155 @@
+            name="name"
+            required
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            min="1" max="100"
+            value="12"
+            autocomplete="off"
+            name="years-old"
+            required
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+            type="date"
+            required
+            autocomplete="off"
+            name="birthday"
+          >
+        </label>
+        <label class="label">
+          I accept the term of the agreement
+          <input type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            name="email"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            autocomplete="off"
+            minlength="12"
+            maxlength="16"
+            name="password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="label">
+          Do you love cats?
+
+          <label>
+            <input type="radio" name="radio-yes">
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="radio-no">
+            No
+          </label>
+        </label>
+
+        <label class="label">
+          What is your favorite color?
+          <input name="color" type="color">
+        </label>
+
+        <label class="label">
+          What time do you go to bad?
+          <input
+            type="time"
+            name="time"
+            step="2"
+          >
+        </label>
+
+        <label class="label" for="cars">What are your favorite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="saab">SAAB</option>
+          </select>
+        </label>
+
+        <label class="label">
+          How do you rate our work?
+          <input
+            name="feedback"
+            type="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="label">
+          Comments:
+          <textarea name="comments" rows="2"></textarea>
+        </label>
+
+        <label class="label" for="rec">Would you recommend us?
+          <select
+            name="recommending"
+            id="rec"
```

Comment:
It's not a good practice to use such short names for ids or classes, it's not clear what does it mean